tests/boulder-integration.sh

```diff
@@ -20,6 +20,18 @@ else
   readlink="readlink"
 fi
 
+cleanup_and_exit() {
+    EXIT_STATUS=$?
+    if SERVER_STILL_RUNNING=`ps -q $python_server_pid -o pid=`
```

Comment:
nit: Perhaps we should use `-p` instead of `-q` here. `-p` appears to be part of the [POSIX standard](http://pubs.opengroup.org/onlinepubs/007904975/utilities/ps.html) while `-q` isn't listed here.

I'm suggesting this just in case a contributor tries to run our tests on a non-linux machine.